test/Transforms/slicing-utils.mlir

```diff
@@ -222,13 +222,12 @@ func @slicing_test() {
 // FWDBWD-LABEL: slicing_test_2
 func @slicing_test_2() {
   %c0 = constant 0 : index
-  %c1 = constant 1 : index
   %c2 = constant 2 : index
   %c16 = constant 16 : index
-  loop.for %i0 = %c0 to %c16 step %c1 {
```

Comment:
This change doesn't look relevant to the commit.

Reply:
It is :-) With a loop.for %i0 isn't a valid dimensional identifier for anything inside (only affine.for IVs are).

Reply:
My bad, I missed it was used in the map in the nested `affine.for`.